playwright/e2e/room/invites.spec.ts

```diff
@@ -19,7 +19,10 @@ test.describe("Invites", () => {
         const roomId = await bot.createRoom({ is_direct: true });
         await bot.inviteUser(roomId, user.userId);
         await app.viewRoomByName("Bob");
-        await expect(page.locator(".mx_RoomView")).toMatchScreenshot("Invites_room_view.png");
+        await expect(page.locator(".mx_RoomView")).toMatchScreenshot("Invites_room_view.png", {
+            // Hide the mxid, which is not stable.
+            mask: [page.locator(".mx_RoomPreviewBar_inviter_mxid")],
```

Comment:
A mask may change width if the length (not monospace font) changes, so not sure this will fix it. It should either be on a fixed size block element or you should use `css`

Reply:
bah, good point. Let's have a look at the tree..

Reply:
(The tree did not offer a good candidate for masking so it's css for you, mxid field)